bydbctl/internal/cmd/root.go

```diff
@@ -30,7 +32,33 @@ func NewRoot() *cobra.Command {
 		DisableAutoGenTag: true,
 		Version:           version.Build(),
 		Short:             "bydbctl is the command line tool of BanyanDB",
+		PersistentPreRunE: func(cmd *cobra.Command, args []string) (err error) {
+			viper.SetConfigType("yaml")
+			viper.SetConfigName("config")
+			viper.AddConfigPath("$HOME/.bydbctl")
+			if err = viper.SafeWriteConfig(); err != nil {
+				if os.IsNotExist(err) {
+					err = viper.WriteConfig()
+					if err != nil {
+						return err
+					}
+				}
+			}
+			if err = viper.ReadInConfig(); err != nil {
+				if _, ok := err.(viper.ConfigFileNotFoundError); ok {
+					logger.GetLogger().Fatal().Err(err).Msg("Config file not found")
```

Comment:
You should create a config file if not found.

Reply:
I think I’ve created it in viper.SafeWriteConfig